.github/workflows/guardian-plan-success.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'guardian_plan_success'
+
+on:
+  workflow_call:
+    inputs:
+      init_result:
+        description: 'The result status for the Guardian Plan Init GitHub workflow.'
+        required: true
+        type: 'string'
+      plan_result:
+        description: 'The result status for the Guardian Plan GitHub workflow.'
+        required: true
+        type: 'string'
+
+permissions:
+  contents: 'read'
+  pull-requests: 'write'
+
+# only one plan, per pr should run at a time
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+
+jobs:
+  plan_success:
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: 'Success'
+        if: ${{ inputs.init_result == 'success' && inputs.plan_result == 'success' }}
+        run: |-
+          echo "All plans successful!"
+          exit 0
+
+      - name: 'Failure'
+        if: ${{ inputs.init_result == 'failure' || inputs.plan_result == 'failure' }}
+        run: |-
+          echo "::error ::Init or Plan has one or more failures."
+          exit 1
+
+      - name: 'Skipped Plan Comment'
+        if: ${{ inputs.plan_result == 'skipped' }}
+        uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975 # ratchet:actions/github-script@v6
```

Comment:
Please quote your yaml

Reply:
i dont know how you see this so easily, hahaha

this was copy pasted so ill fix everywhere.